pkg/nodebootstrap/al2_test.go

```diff
@@ -149,7 +167,8 @@ API_SERVER_URL=
 B64_CLUSTER_CA=
 NODE_LABELS=
 NODE_TAINTS=
-CLUSTER_DNS=172.16.0.10`,
+CLUSTER_DNS=172.16.0.10
+CONTAINER_RUNTIME=`,
```

Comment:
i am surprised that this var is empty if we always have that config set on the node (containerd if asked for, dockerd the rest of the time) 🤔  am i reading wrong? did you do a test ng create without that conf option?

Reply:
This test doesn't call defaults it sets it manually so defaults aren't applied to it. I think normally, the set defaults is always called, so I thought it's fine in this test. Do you think it would be better to ~hardcode it~ set it as default somewhere else?

Reply:
aaaah yeh that makes all the sense. nope what you have is fine i think.